SimCalorimetry/EcalEBTrigPrimAlgos/src/EcalFenixTcpFormat.cc

```diff
@@ -121,7 +121,7 @@ void EcalFenixTcpFormat::process(std::vector<int> &Et, std::vector<int> &fgvb,
       else
 	lut_out = (lut_)[myEt] ;
       
-      //int ttFlag = (lut_out & 0x700) >> 8 ;
+      int ttFlag = (lut_out & 0x700) >> 8 ;
```

Comment:
This is not needed: Value stored to 'ttFlag' during its initialization is never read